appeals/e2e/cypress/page_objects/caseDetailsPage.js

```diff
@@ -185,10 +190,27 @@ export class CaseDetailsPage extends Page {
 		this.elements.costDecisionStatus().contains(text);
 	}
 
+	clickRemoveFileUpload(buttonIndex) {
+		this.elements.removeFileUpload(buttonIndex).click();
+	}
+
 	/***************************************************************
 	 ************************ Verfifications ************************
 	 ****************************************************************/
 
+	checkErrorMessageDisplays(errorMessage) {
```

Comment:
for error message could you use 'validateErrorMessage' function on base page?

Reply:
I've tried that but it appears the elements I'm testing uses a different class-name